Meetings/2022-03-09-meeting-minutes.md

```diff
@@ -0,0 +1,73 @@
+# Maintainers' Meeting 2022-03-09
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Daniel Valadas - @valadas
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- Halil İbrahim Kalkan @hikalkan
+- James Turner - @Turnerj
+- Michael Hawker - @michael-hawker
+- Nils Andresen - @nils-a
+- Pascal Berger - @pascalberger
+- Robin Krom - @Lakritzator
+
+## Recording
+
+[2022-03-09 Meeting Recording](https://dotnetfoundation-my.sharepoint.com/:v:/g/personal/rob_prouse_dotnetfoundation_org/EYdWzSDu0qVPkN8vPRw6Ux8BuF8SMfimpSpWlCwCVxa3QA?e=KFvGBC)
+
+### Virtual Inc Staff Observers
+
+- Tom Pappas
+- Nicole Miller
+
+## Slides
+
+## Agenda
+
+- Review previous [meeting minutes](https://github.com/dotnet-foundation/wg-maintainers/blob/main/Meetings/2022-02-16-meeting-minutes.md)
+- Briefly discuss ()[https://github.com/dotnet-foundation/wg-maintainers/issues/48]
+- Talk about how we can provide better guidance to maintainers from this thread as well: [Centralised information for .NET Foundation operation](https://github.com/dotnet-foundation/Home/discussions/61)
+- Discuss #55 and #56
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Process for leaving the foundation
+  - Board has voted to approve the process to leave the Foundation
+  - Tom Pappas met with the lawyers to review the process and to review the agreements that we will use going forward.
+  - We should put forward an FAQ for each agreement for joining the foundation
+  - Project committee will then review, then go to the Executive Committee an the Board
+  - Hoping to start July 1 with the new agreements and a clearer process for projects joining the foundation.
+- Want to have a joint meeting of the maintainers and project committees to help define our roles.
+- Project benefits PR, is being worked on and moving forward
+- Tom Pappas took the role of Executing Director Feb 1, will be announced in the next newsletter
+- Brief discussion about the probationary committee membership
```

Comment:
It was mentioned that no social media information would be exposed until active participants in the DNF maintainers committee.